tasks/main.yml

```diff
@@ -35,7 +35,18 @@
   when: "ansible_service_mgr == 'systemd'"
 
 - name: Ensure the firewall is enabled and will start on boot.
-  service: name=firewall state=started enabled=yes
+  service:
+    name: firewall
+    state: started
```

Comment:
Instead of adding a 2nd `service` task, could you switch this to do something like the following?

```yaml
  - name: Ensure the firewall is in the correct state.
    service:
      name: firewall
      state: {{ firewall_state }}
      enabled: {{ firewall_enabled_at_boot }}
```

Then we could default those two variables to `started` and `yes`, respectively, and we can keep this task nice and simple. This would also allow more flexibility if someone wanted to just have the firewall disabled until a reboot, or have it enabled but not on boot.